cmd/cluster.go

```diff
@@ -1,6 +1,8 @@
 package cmd
 
 import (
+	"errors"
+	"fmt"
```

Comment:
Add empty line between std lib package and external
```suggestion
	"fmt"

```